### DISCUSSION TITLE
sew_resnet and spiking_resnet

### DISCUSSION THREAD

**Discussion by cameron-git** [fangwei123456/SpikingFlow#409] [Q&A] [answered]
In the [Train Large-Scale SNN](https://spikingjelly.readthedocs.io/zh_CN/0.0.0.0.8/clock_driven_en/16_train_large_scale_snn.html) tutorial the spiking resnet is used but in [Deep Residual Learning in Spiking Neural Networks](https://arxiv.org/pdf/2102.04159.pdf) it seems inferior than sew resnet especially for deeper networks.
Is the spiking resnet implementation the same as the one in the paper?
Why was sew resnet not chosen?
For resnet18 the accuracy seems similar, is there a difference in training or inference time?

Thank you in advance

**Comment by fangwei123456** [comment, marked as answer]
> Is the spiking resnet implementation the same as the one in the paper?

Yes.

> Why was sew resnet not chosen?

Because it requires the prior knowledge about what is sew resnet, which is not friendly for new users.

> For resnet18 the accuracy seems similar, is there a difference in training or inference time?

The difference is the same with that in PyTorch, such as the behavior of BN is not the same in training and inference.
